common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/WebViewAuthorizationFragment.java

```diff
@@ -115,13 +117,25 @@ public class WebViewAuthorizationFragment extends AuthorizationFragment {
 
     private PermissionRequest mCameraPermissionRequest;
 
+    private ActivityResultLauncher<LegacyFido2ApiObject> mFidoLauncher;
```

Comment:
nit: add a comment that this would be invoked (required) by the legacy fido class?

Reply:
be71af20c85ba1d393b2f85e273871c29b9bdcef